api-reference/10 UI Widgets/dxFileManager/5 File System Providers/FileSystemProviderBase/3 Methods/abortFileUpload().md

```diff
@@ -3,19 +3,18 @@ id: FileSystemProviderBase.abortFileUpload()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Cancels the file upload.
 
 ##### return: Promise<any>
-<!-- Description goes here -->
+A Promise that is resolved after the file upload in aborted. It is a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">native Promise</a> or a <a href="http://api.jquery.com/Types/#Promise" target="_blank">jQuery.Promise</a> when you use jQuery.
 
 ##### param(fileData): File
-<!-- Description goes here -->
+The file that is uploaded.
 
 ##### param(uploadInfo): Object
-<!-- Description goes here -->
+Information about the file upload session.
 
 ##### param(destinationDirectory): FileSystemItem
-<!-- Description goes here -->
+The directory where a file system item is uploaded to.
```

Comment:
If these are arguments passed to the method, they should be listed in the method's ID:
```
id: FileSystemProviderBase.abortFileUpload(fileData, uploadInfo, destinationDirectory)
```
This change should be done in the doccomments.

The same applies to all methods that have arguments.

Reply:
ok, I sent this to the programmer.

Reply:
this will be fixed next 1-2 weeks